schema/resource.go

```diff
@@ -101,6 +101,18 @@ func (r *Resource) Columns() []string {
 	return r.Table.Columns.Names()
 }
 
+// Validates various constraints. For example: Does all primary keys has values.
```

Comment:
```suggestion
// Validates that all primary keys have values.
```

I believe the comment should reflect what the function does now. We can change the comment if/when we add more functionality